pytorch_lightning/loops/base.py

```diff
@@ -46,6 +46,15 @@ class Loop(ABC):
     def __init__(self) -> None:
         self.iteration_count: int = 0
         self.trainer: Optional['pl.Trainer'] = None
+        self._restarting = False
+
+    @property
+    def restarting(self) -> bool:
+        return self._restarting
+
+    @restarting.setter
+    def restarting(self, restarting: bool) -> None:
+        self._restarting = restarting
```

Comment:
Is there a reason why you added a getter/setter considering they don't do anything custom?

Reply:
Not a strong need for them right now. Can be removed later on.